nixos/modules/services/mail/dovecot.nix

```diff
@@ -209,7 +210,7 @@ let
 
   modulesDir = pkgs.symlinkJoin {
     name = "dovecot-modules";
-    paths = map (pkg: "${pkg}/lib/dovecot") (
+    paths = map (pkg: "${pkg}/etc/dovecot/modules") (
```

Comment:
Modules do not belong in /etc, they are not configuration data. Keep them in /lib in the packages.

Reply:
Yes, but then where to load all modules from?

Reply:
Fixed!